modules/swagger-codegen/src/main/resources/python/README.mustache

```diff
@@ -128,3 +128,28 @@ Class | Method | HTTP request | Description
 {{/isOAuth}}
 
 {{/authMethods}}
+
+## Integration with Google Cloud Platform
+
+Swagger client is using urllib3 as default http library, however Google Cloud Platform disallow it. 
+There are two ways you can use urllib3 in Appengine Standard: patch it to use App Engine’s URLFetch or enable sockets. 
+Which one to use depends on your situation and needs. URLFetch is more cost-effective as long as your usage is within the limitations for free app. Sockets are only available for paid apps and are in beta at the time of writing.
```

Comment:
Is that still true? Maybe we should remove the last 3 sentences, and replace them with a link to this doc: https://urllib3.readthedocs.io/en/latest/reference/urllib3.contrib.html#module-urllib3.contrib.appengine

Reply:
as @alavers mentioned to me, links tends to go dead so its better to put the text directly in. It would, however, probably be more simple to just put the url I agree.

Reply:
:+1: I think we should remove ` and are in beta at the time of writing` though (at least it's not mentioned here: https://cloud.google.com/appengine/docs/standard/python/sockets/)

Reply:
Removed!